tailwind.config.js

```diff
@@ -0,0 +1,70 @@
+// tailwind.config.js
+
+import tailwindcssAnimate from "tailwindcss-animate";
+
+/** @type {import("tailwindcss").Config} */
+module.exports = {
+  content: ["./src/**/*.{js,ts,jsx,tsx}", "./src/styles/tailwind.css"],
+  darkMode: ["class", '[data-theme="dark"]'],
+  plugins: [tailwindcssAnimate],
+  corePlugins: {
+    preflight: false,
+  },
+  theme: {
+    extend: {
```

Comment:
can we not extend but redefine all the tokens? Otherwise, people can still use predefined tailwind tokens without realizing that it's not connected to obsidian CSS variables. We can also turn on this eslint rule to enforce only supported classnames are used in the code base: https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/no-custom-classname.md